src/components/ProductFilters.tsx

```diff
@@ -1,103 +1,117 @@
-import { ChangeEvent, FC, useEffect, useState } from "react";
+import { ChangeEvent, FC, useState } from "react";
 import { useAppDispatch, useTypedSelector } from "../store";
 import {
-  filterProductsByPrice,
-  filterProductsBySize,
-  sortProductsByType,
-  updateChosenSizes,
-} from "../store/slices/productList.slice";
+  setSizeFilter,
+  setPriceFilter,
+  setSortType,
+  sortByTypeAction,
+  filterByPriceAction,
+  filterBySizeAction,
+} from "../store/slices/productFilters";
 import { setGloablLoading } from "../store/slices/global.slice";
 import Form from "react-bootstrap/Form";
-import { Col, Container } from "react-bootstrap";
+import { Button, Col, Container, Row } from "react-bootstrap";
+import { singleProduct } from "../mock-data";
+import { productSortTypes } from "../store/slices/productFilters/model";
 
-const selectSortOptions = [
-  { type: "low_First", name: "low price first" },
-  { type: "high_First", name: "high price first" },
-];
 export type ProductFiltersProps = {};
 
 const ProductFilters: FC<ProductFiltersProps> = () => {
-  const filter = useTypedSelector((state) => state.productList.productFilters);
-  const [maxPrice, setMaxPrice] = useState<number>(filter.price.max);
   const dispatch = useAppDispatch();
+  const {filterOptions} = useTypedSelector((state) => state.productFilters);
+  const [activeButton, setActiveButton] = useState<boolean>(false);
+  const [maxPrice, setMaxPrice] = useState<number>(100);
```

Comment:
move min-max to static data and get values from there (useState unnecessary)

Reply:
I don't fully understand the remark.
These states store the value of inputs (in the future I would like to add a pointer for the minimum price). Then, with a mouse up, the value is transferred to the dispatcher and stored in the store.
You may have thought about these values that we specify for the maximum and minimum input values. But I hardcoded them in the <Form.Range component
             min={1}
             max={100}